src/api/store/entity.ts

```diff
@@ -63,7 +47,7 @@ export type Menu = SinglePriceMenu | MultiPriceMenu;
 
 export interface MenuCategory {
   id: number;
-  name: string;
+  name: '이벤트 메뉴' | '대표 메뉴' | '추천 메뉴 ' | '사이드 메뉴' | '세트 메뉴';
```

Comment:
카테고리 변경이 가능하게끔 api가 구성돼있던거같은데, 요거는 이제 deprecated된걸까요?
그리고 추천메뉴에 오타 껴있어요!
구성이 확정이라면 Type으로 만들어두는게 좋아보이네용 
```suggestion
  name: '이벤트 메뉴' | '대표 메뉴' | '추천 메뉴' | '사이드 메뉴' | '세트 메뉴';
```